src/i18_bluesky/plans/simple_plans.py

```diff
@@ -0,0 +1,88 @@
+from typing import Any, List, Mapping, Optional
+
+import bluesky.plans as bp
+from dls_bluesky_core.core import MsgGenerator
+from ophyd_async.core.detector import StandardDetector
+from ophyd_async.epics.motion import Motor
+
+"""
+Dictionary of scan arguments from motor scan_args (start, stop, steps)
+"""
+
+
+def make_args(motor, scan_args, prefix=""):
+    return {
+        "motor" + prefix: motor,
+        "start" + prefix: scan_args[0],
+        "stop" + prefix: scan_args[1],
+        "steps" + prefix: int(scan_args[2]),
+    }
+
+
+def step_scan(
+    detectors: List[StandardDetector],
+    motor: Motor,
+    scan_args: List[object],
+    metadata: Optional[Mapping[str, Any]] = None,
+) -> MsgGenerator:
```

Comment:
```suggestion
def step_scan(
    detectors: List[StandardDetector],
    motor: Motor,
    *scan_args: tuple[Motor, float, float, int],
    *,
    metadata: Optional[Mapping[str, Any]] = None,
) -> MsgGenerator:
```

As I understand it, this makes scan_args take an arbitrary number of tuples of that form, has metadata as kw-only argument.